NutriFit-aims-objectives.md

```diff
@@ -0,0 +1,41 @@
+## Aims and Objectives
+
+### Project Aim
+<p>The aim of the NutriFit app project is to design and develop a user-centric, web-based application.The application 
+will integrate fitness tracking with personalised recipe recommendations thereby empowering users to lead healthier
+lifestyles.The application will have fitness and recipe monitoring tools for fitness progress, meal planning based on 
+dietary needs and preferences to achieve fitness and nutrition goals logically.</p>
+
+## Project Objectives
+### 1.  Development Concept
+
++ Develop the application  using the MEAN (MongoDB(PostgreSQL), Angular, and Node.js) stack for 
```

Comment:
Maybe alter to PEAN ?
I don't see the value of using MEAN if we're not going to use Mongo.

Also don't forget the E for Express.js, that's the librairie used to handle the webserver on node.

Reply:
We're using PostGreSQL correct ? You've set it as MongoDB on the file ?

Reply:
Sorry, my mistakes. Yes we are using PostGreSQL.  I will make the correction now. 
Thanks!

Reply:
Is everything okay now? If so, I can go ahead and merge.

Reply:
Yes, all good 👍

Reply:
Great! I merge right away and delete the branch.
Thanks!